### ISSUE TITLE
Switch bg props not working

### ISSUE THREAD

**Issue by pierroo** [tamagui/tamagui#2413]
### Current Behavior

The Switch component is supposed to have a working "bg" props, but it doesn't work at all.
Used to work in v1.43.12, I just upgraded to latest tamagui 1.91.3, and now the bg is just ignored.

### Expected Behavior

bg props should work on Switch

### Tamagui Version

```markdown
1.91.3
```

### Platform (Web, iOS, Android)

```markdown
ios & android
```

### Reproduction

```markdown
<Switch
                            size="$3"
                            checked={e.isChecked}
                            onCheckedChange={e.toggleSwitch}
                            bg={e.isChecked ? GREEN : 'rgba(255,255,255,.15)'}
                        >
                            <Switch.Thumb animation="bouncy" bg={"white"} />
                        </Switch>
```

**Comment by pierroo** [comment]
I am sorry but, even using your latest release 1.91.4 the bg props on Switch still isn't working (switch thumb is, though) @TheEhsanSarshar

**Comment by TheEhsanSarshar** [comment]
Actually that's not released yet, should be available in `1.91.5`
